### ISSUE TITLE
Try cervical dorsal model on whole spine image

### ISSUE THREAD

**Issue by valosekj** [ivadomed/model-spinal-rootlets#57]
When we tried the cervical model ([r20240523](https://github.com/ivadomed/model-spinal-rootlets/releases/tag/r20240523)) segmenting _both_ dorsal and ventral rootlets on a whole spine image, the model predicted some false positive rootlets in the Th/L area; details in [this comment](https://github.com/ivadomed/model-spinal-rootlets/pull/53#issuecomment-2179457525). It would also be interesting to see prediction from the dorsal model, which is part of the SCT (`sct_deepseg -task seg_spinal_rootlets_t2w`). 
@KaterinaKrejci231054, can you please run the model on the same subject to see the prediction? Thank you!

**Comment by valosekj** [comment]
Thank you for running the model! It's great to see that `sct_deepseg -task seg_spinal_rootlets_t2w` does not segment any false positive rootlets below the C8 rootlets! @joshuacwnewton, do you know if `sct_deepseg` applies any post-processing after running the inference? For example, removing small objects, etc.?

**Comment by joshuacwnewton** [comment]
Yes! We've just added support for using the existing sct_deepseg postprocessing options (-remove-small, -largest) on monai/nnunetv2 models. 

(Before, they only worked for ivadomed models)
